src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -263,6 +263,25 @@ enum AlertSeverityLevelType {
     SEVERE
 }
 
+type OpeningHours {
+    """
+    OSM-formatted string of the opening hours.
+
+    The spec is available at: https://wiki.openstreetmap.org/wiki/Key:opening_hours
+    """
+    osm: String
+    """
+    Opening hours for the selected dates using the local time of the parking lot.
+    Each date can have multiple time spans.
```

Comment:
Not sure if we should add a mention here that this is not yet(?) implemented.

Reply:
Added comment